test/cljam/io/bcf/writer_test.clj

```diff
@@ -267,3 +267,16 @@
          0x17 0x43
          0x00]
         []]])))
+
+(deftest default-meta-info
+  (let [tmp (File/createTempFile "default-meta-info" ".bcf")
+        header ["CHROM" "POS" "ID" "REF" "ALT" "QUAL" "FILTER" "INFO"]
+        s (->> ["##fileformat=VCFv4.3"
+                "##FILTER=<ID=PASS,Description=\"All filters passed\",IDX=0>"
+                (str "#" (cstr/join \tab header) \newline (char 0))]
+               (cstr/join \newline))
+        _ (with-open [w (bcf-writer/writer tmp {} header)])
```

Comment:
`w` is an unused binding. Please use underscore.  https://github.com/bbatsov/clojure-style-guide#underscore-for-unused-bindings

Reply:
Thanks! Fixed in 93d87ee